driver/CMakeLists.txt

```diff
@@ -131,31 +105,39 @@ if(USE_SSL)
         PUBLIC ${Poco_NetSSL_LIBRARY}
         PUBLIC ${Poco_Crypto_LIBRARY}
     )
-    target_include_directories (${libname}-impl PUBLIC ${OPENSSL_INCLUDE_DIR})
+    target_include_directories (${libname}-impl
+        PUBLIC ${OPENSSL_INCLUDE_DIR}
+    )
 endif()
 
 target_link_libraries(${libname}-impl
     PUBLIC ${Poco_Net_LIBRARY}
     PUBLIC ${Poco_Foundation_LIBRARY}
     PUBLIC ${Poco_Util_LIBRARY}
     PUBLIC ${ODBC_LIBRARIES}
+    PUBLIC Threads::Threads
 )
 
-if(WIN32)
-    target_link_libraries(${libname}-impl PUBLIC ws2_32 Iphlpapi legacy_stdio_definitions)
-endif()
+if (WIN32)
+    target_link_libraries(${libname}-impl PUBLIC Ws2_32 Iphlpapi)
+endif ()
 
-if(ARCH_LINUX)
-    target_link_libraries(${libname}-impl PUBLIC Threads::Threads)
-endif()
+add_library(${libname} SHARED
```

Comment:
Not sure, what do you mean here. You say, why we have `*_impl` targets at all, instead of just linking the resulting odbc driver '.so' with test executables? (That was originally introduced by you, btw.)

Reply:
I mean here you have a shared library with some extra sources that can't be tested by unit test binaries if some symbols are not public (exported).

Reply:
Those files contain (or rather should contain) only ODBC APL functions that forward the call to the corresponding `impl::F()` functions (I will gradually do this as a part of the next iterations of `api-cleanup`). Not sure, how you could cleanly test ODBC calls bypassing Driver Manager. However, when all actual implementations are factored out into `impl::F()`-like functions, unit tests could call them, while the actual ODBC API calls will be trivial forwarders to those.

Reply:
Ok